packages/react-components/react-link/src/components/Link/useLinkState.ts

```diff
@@ -8,12 +8,12 @@ import type { LinkState } from './Link.types';
  */
 export const useLinkState_unstable = (state: LinkState): LinkState => {
   const { disabled, disabledFocusable } = state;
-  const { onClick, onKeyDown, role, type } = state.root;
+  const { onClick, onKeyDown, role, type, tabIndex } = state.root;
```

Comment:
_nit:_ Maintain alphabetical order

```suggestion
  const { onClick, onKeyDown, role, tabIndex, type} = state.root;
```